builder/src/scss/general.scss

```diff
@@ -58,6 +58,21 @@
     }
 }
 
+.left-main-column,
+.right-main-column {
+    flex-grow: 1;
+    flex-basis: 0;
+}
+#left-main-column,
+#right-main-column {
+    height: 100%;
+}
+.wrapper-column-container {
```

Comment:
You also might want to add `width: 100%` here just to make sure the width is passed down properly